go/cmd/berty/main.go

```diff
@@ -418,11 +422,107 @@ func main() {
 		},
 	}
 
+	shareInvite := &ffcli.Command{
```

Comment:
Shouldn't be more relevant to add this feature in the app? (Maybe bound to your go package though a gRPC call)
I'm not sure to understand the use case of this command.

Reply:
yes I want to move it to the app, but since it's a new devtool I want to merge this V1 first then talk with @n0izn0iz and @gfanton about how they prefer that we implement this

btw, this command is for sharing your local QR code (the one used with berty mini or a daemon), and it's not limited to sending to discord, actually, it's an opt-in option (cf screenshots)